share/completions/pygmentize.fish

```diff
@@ -1,8 +1,12 @@
 function __fish_print_pygmentize
-    set -l lines (pygmentize -L $argv[1] | string match -r '^(?:\* |    ).*(?:)$' | string replace -r '\* (.*):$' '$1' | string replace -r '^(.*).$' '$1' | string trim)
+    set -l lines (pygmentize -L $argv[1] | string match -r '^(?:\* |    ).*(?:)$' | string replace -r '\* (.*):$' '$1' | string replace -r '^(.*)^\.' '$1' | string trim)
```

Comment:
My bad, I made a (other) mistake while trying to fix an issue. Basically, what it was suppose to do, it to remove the period at the end of the descriptions... and I first forgot to escape the `.` (so it removed the last characters instead).

So it should be `string replace -r '^(.*)\.$' '$1'` instead of this mess.

```
> pygmentize -L filters 
Pygments version 2.1.3, (c) 2006-2015 by Georg Brandl.

Filters:
~~~~~~~~
* highlight:
    Highlight a normal Name (and Name.*) token with a different token type.
* whitespace:
    Convert tabs, newlines and/or spaces to visible characters.
* gobble:
    Gobbles source code lines (eats initial characters).
* raiseonerror:
    Raise an exception when the lexer generates an error token.
* keywordcase:
    Convert keywords to lowercase or uppercase or capitalize them, which means first letter uppercase, rest lowercase.
* codetagify:
    Highlight special code tags in comments and docstrings.
* tokenmerge:
    Merges consecutive tokens with the same token type in the output stream of a lexer.
```